pipeline-model-extensions/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/agent/DeclarativeAgent.java

```diff
@@ -66,6 +73,25 @@ public boolean isDoCheckout() {
         return doCheckout;
     }
 
+    public void addOption(@Nonnull DeclarativeOption option) {
```

Comment:
Is there any effect to this? I can't see any.

Reply:
Nope, I just left it dangling. On second thought, I'll clean it out.